bats/tests/containers/factory-reset.bats

```diff
@@ -116,15 +116,15 @@ check_installation() {
         echo "$assert that $profile does not add ~/.rd/bin to the PATH"
         # cshrc: setenv PATH "/Users/jan/.rd/bin"\:"$PATH"
         # posix: export PATH="/Users/jan/.rd/bin:$PATH"
-        run grep "PATH.\"$HOME/.rd/bin" $profile
+        run grep "PATH.\"$HOME/.rd/bin" "$profile"
         ${assert}_failure
```

Comment:
It doesn't have to because it can only have the values `assert` or `refute`. Shellcheck "knows" this; it would tell you that the variable must be quoted if it cannot figure out all the possible values for it.

I'll add quotes here too, it doesn't hurt.

Reply:
Actually, I changed my mind. `shellcheck` "knows" that there aren't spaces in the value. 

And even if there were spaces in the name, it would still be a syntax error, even if we quoted it, as you cannot have spaces in command or function names. So adding quotes would just be noise.